commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -300,33 +300,34 @@ def get_inventory_from_file(environment):
     state = None
     with open(environment.paths.inventory_ini) as f:
         for line in f.readlines():
-            group_line_match = re.match(r'^\[\s*(.*)\s*\]\s*$', line)
-            if re.match(r'^\s*$', line):
-                continue
-            if group_line_match:
-                section_name = group_line_match.groups()[0]
-                if section_name.endswith(':children'):
-                    state = 'parsing-group'
-                    current_group_name = section_name[:-len(':children')]
-                    current_group = Group(name=current_group_name)
-                    inventory.all_groups[current_group_name] = current_group
-                else:
-                    state = 'parsing-host'
-                    current_host_name = section_name
-            else:
-                if state == 'parsing-host':
-                    line_groups = list(re.split(r'\s+', line.strip()))
-                    private_ip, variables = line_groups[0], line_groups[1:]
-                    variables = dict(var.strip().split('=') for var in variables)
-                    public_ip = variables.pop('ansible_host')
-                    host = Host(name=current_host_name, private_ip=private_ip, public_ip=public_ip,
-                                vars=variables)
-                    inventory.all_hosts.append(host)
-                elif state == 'parsing-group':
-                    host_name = line.strip()
-                    current_group.host_names.append(host_name)
+                group_line_match = re.match(r'^\[\s*(.*)\s*\]\s*$', line)
+                if re.match(r'^\s*$', line):
+                    continue
+                if group_line_match:
+                    section_name = group_line_match.groups()[0]
+                    if section_name.endswith(':children'):
+                        state = 'parsing-group'
+                        current_group_name = section_name[:-len(':children')]
+                        current_group = Group(name=current_group_name)
+                        inventory.all_groups[current_group_name] = current_group
+                    else:
+                        state = 'parsing-host'
+                        current_host_name = section_name
                 else:
-                    raise ValueError('Encountered items outside a section')
+                    if state == 'parsing-host':
+                        if 'ansible_host' in line:
```

Comment:
I think this is the other change. Wasn't this a change from the PR we decided not to make?

Reply:
I just reverted 2 commits which I accidentally pushed on this branch:
https://github.com/dimagi/commcare-cloud/pull/1879/commits/ae91419f29e9994ba480baefa47975ef87c6ea06
https://github.com/dimagi/commcare-cloud/pull/1879/commits/9d186dfab6cfb6c3241c8e0cfc6eef798a68d82a